runtime/autoload/remote/host.vim

```diff
@@ -118,7 +118,7 @@ function! remote#host#RegisterPlugin(host, path, specs) abort
 endfunction
 
 
-function! s:GetInitScript() abort
+function! s:GetManfiest() abort
```

Comment:
No no.  It's right and totally not a typo that was aided by completion and sleepiness.